gallery/hci-insights/microsoft.azurestackhci-clusters.json

```diff
@@ -17,6 +17,18 @@
 					"author": "Microsoft",
 					"description": "Performance, health and usage of your registered Azure Stack HCI cluster that are connected to Azure and are enrolled in monitoring",
 					"name": "Insights"
+				},
+				{
+					"id": "Workbooks/Azure Stack HCI/SingleClusterAMAPreview",
```

Comment:
This is the overview workbook right? Why are we naming it singleClusterAMAPreview?

Reply:
Yes. Its the overview workbook only. Just using a different template name to address the overview workbook.